doc/index.rst

```diff
@@ -2,21 +2,21 @@ CV32E20: An embedded 32-bit RISC-V CPU core
 ===========================================
 
 CV32E20 is a production-quality open source 32-bit RISC-V CPU core written in SystemVerilog.
-The CPU core is based on the Ibex core, but simplified and verified under the OpenHW Group.
+The CPU core is based on the CVE2 core, but simplified and verified under the OpenHW Group.
```

Comment:
No, it is still true that the CV32E20 is based on Ibex, so we should leave this one as is.  If you like, you could update it to say this:
```
CV32E20 is a specific configuration of the CVE2 and is a production-quality open source 32-bit RISC-V CPU core written in SystemVerilog.
The CVE2 is based on the lowRISC Ibex core, but simplified and (re) verified by the OpenHW Group.